src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderTransactionHistoryDto.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Builder;
+import lombok.ToString;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import java.time.LocalDate;
+
+@ApiModel(description = "Offender transaction details")
+@Data
+@EqualsAndHashCode
+@ToString
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class OffenderTransactionHistoryDto {
```

Comment:
This model is missing a currency property.

Reply:
A small thing but we don't generally use the Dto suffix on the API model - it's implied.

Reply:
removing Dto will result in a fully qualified class name in the service class uk.gov.justice.hmpps.prison.api.model.OffenderTransactionHistory as the entity has the same name uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderTransactionHistory, is that okay?

Reply:
Good point.  Yes that's been done previously so it is okay but I understand if you want to avoid having to include package names in code.  If only we were using Kotlin :)

Reply:
done